packages/components/src/mobile/keyboard-avoiding-view/index.ios.js

```diff
@@ -3,18 +3,75 @@
  */
 import {
 	KeyboardAvoidingView as IOSKeyboardAvoidingView,
+	Animated,
+	Keyboard,
 	Dimensions,
 } from 'react-native';
 
-export const KeyboardAvoidingView = ( { parentHeight, ...otherProps } ) => {
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect, useRef } from '@wordpress/element';
+
+const AnimatedKeyboardAvoidingView = Animated.createAnimatedComponent(
+	IOSKeyboardAvoidingView
+);
+
+const MIN_HEIGHT = 44;
+
+export const KeyboardAvoidingView = ( {
+	parentHeight,
+	style,
+	withAnimatedHeight = false,
+	...otherProps
+} ) => {
+	const [ keyboardHeight, setKeyboardHeight ] = useState( 0 );
+	const animatedHeight = useRef( new Animated.Value( 44 ) ).current;
+
 	const { height: fullHeight } = Dimensions.get( 'window' );
```

Comment:
```suggestion
	const fullHeight = useWindowDimensions().height;
```

Reply:
FYI I tried to use `useWindowDimensions` a while back and encountered a [multiple render issue](https://github.com/facebook/react-native/issues/26733), that is fixed in RN 0.62 but we haven't finished updating yet. Thought I should mention it just in case you see the same.